zanlib/classes/zandronum/rcon/ZandronumRcon.cs

```diff
@@ -0,0 +1,39 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Zanlib
+{
+    public partial class ZandronumRcon
+    {
+        
+        private readonly NetworkHelper networkHelper;
+
+        public ZandronumRcon(string ServerName = "localhost", int Port = 10666)
+        {
+            networkHelper = new NetworkHelper(ServerName, Port);
+
+            Iwad = new Iwad(networkHelper);
+            Limits = new Limits(networkHelper);
+            MapName = new MapName(networkHelper);
+            MaxPlayers = new MaxPlayers(networkHelper);
+            Name = new Name(networkHelper);
+            Players = new Players(networkHelper);
+            PWads = new Pwads(networkHelper);
+            Skill = new Skill(networkHelper);
+            Teams = new Teams(networkHelper);
```

Comment:
I don't feel that changing these to be object instances and putting them in another class adds any value aside from "object orienting" the code.